src/config/config.json

```diff
@@ -0,0 +1,13 @@
+{
+    "baseUrl": "http://localhost:8081",
```

Comment:
should be https://node1.testnet.pokt.network

Reply:
or maybe we need two config.json files and the one that is used is determined by whether you start dev or prod